lib/src/play_store_search_api.dart

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2021 William Kwabla. All rights reserved.
+ */
+
+import 'package:html/parser.dart' show parse;
+import 'package:http/http.dart' as http;
+
+class PlayStroeSearchApi {
```

Comment:
@Williano The word stroe should be store.

Reply:
To follow the same conventions as `ITunesSearchAPI`, it should be named `PlayStoreSearchAPI`.